app/assets/javascripts/nilavu/controllers/topic-predeploy.js.es6

```diff
@@ -39,6 +43,7 @@ export default Ember.Controller.extend({
 
         if (this.deployError) {
             this.set('barLabel', "Ooops, try relaunching...");
+            this.set('err', this.get('style'));
```

Comment:
replace with `bind-attr` and flip that variable here ? Just so the css sets automatically  ?